src/main/java/com/dreamsportslabs/guardian/constant/Constants.java

```diff
@@ -82,4 +82,7 @@ public final class Constants {
 
   public static final String ACCESS_TOKEN_COOKIE_NAME = "AT";
   public static final String REFRESH_TOKEN_COOKIE_NAME = "RT";
+
+  public static final int PAGE = 1;
+  public static final int MAX_PAGE_SIZE = 500;
```

Comment:
Keep Page configs specific to the API please, since there are other APIs that are paginated as well. 
Suggestion- either move these constants to the respective files they are used in, or prefix the constant names to reflect where they are used as page is too generic